src/api/router.py

```diff
@@ -48,3 +52,35 @@ async def get_tasks_for_user(user_id: int, task_service: TaskService = Depends()
 )
 async def get_all_tasks(task_service: TaskService = Depends()) -> list[TaskResponse]:
     return await task_service.get_all()
+
+
+@api_router.get(
+    "/telegram/feedback_form",
```

Comment:
Нижнии подчеркивания в url не используют

```suggestion
    "/telegram/feedback-form",
```